### ISSUE TITLE
Return type of `fmi2GetNames`

### ISSUE THREAD

**Issue by sathvikbhagavan** [ThummeTo/FMIImport.jl#86]
`fmi2GetNames` returns a vector of Any. It should return a vector of strings as they are list of names. This affects for simulating as well.

MWE:

```julia
using FMI
using FMIImport

fmu = FMI.fmi2Load("CoupledClutches.fmu")
output_names = FMIImport.fmi2GetOutputNames(fmu)
sol = FMI.fmi2Simulate(fmu; recordValues=output_names)
```

gives an error:

```julia
ERROR: TypeError: in keyword argument recordValues, expected Union{Nothing, Int64, UInt32, AbstractVector{Int64}, AbstractVector{String}, AbstractVector{UInt32}, String, Symbol}, got a value of type Vector{Any}
```

Here `output_names` should return a vector of strings

**Comment by ThummeTo** [comment]
Jap, thanks for the fix!

-----------------

Offtopic, but you could also do:
```
using FMI

fmu = FMI.fmi2Load("CoupledClutches.fmu")
sol = FMI.fmi2Simulate(fmu; recordValues=:outputs) # wildcard symbol for all output value references
```

**Comment by ThummeTo** [comment]
fixed in 0.15.4
